safe_eth/eth/clients/blockscout_client.py

```diff
@@ -135,8 +135,8 @@ class BlockscoutClient:
 
     def __init__(self, network: EthereumNetwork):
         self.network = network
-        self.grahpql_url = self.NETWORK_WITH_URL.get(network)
-        if self.grahpql_url is None:
+        self.grahpql_url = self.NETWORK_WITH_URL.get(network) or ""
+        if not self.grahpql_url:
```

Comment:
```suggestion
        self.grahpql_url: Optional[str] = self.NETWORK_WITH_URL.get(network)
        if self.grahpql_url is None:
```

Reply:
Isn't this working?

Reply:
Ok, then `self.grahpql_url = self.NETWORK_WITH_URL.get(network, "")` would be better 😉

Reply:
Yes, I already updated this in the commit that I mentioned above :)